src/distrs/binom.jl

```diff
@@ -1,19 +1,25 @@
 # functions related to binomial distribution
 
 import .RFunctions:
-    binompdf,
-    binomlogpdf,
-    binomcdf,
-    binomccdf,
-    binomlogcdf,
-    binomlogccdf,
+    # binompdf,
+    # binomlogpdf,
+    # binomcdf,
+    # binomccdf,
+    # binomlogcdf,
+    # binomlogccdf,
     binominvcdf,
     binominvccdf,
     binominvlogcdf,
     binominvlogccdf
 
-# pdf for numbers with generic types
-binompdf(n::Real, p::Real, k::Real) = exp(binomlogpdf(n, p, k))
+binompdf(n::Real, p::Real, k::Real) = betapdf(k + 1, n - k + 1, p)/(n + 1)
 
-# logpdf for numbers with generic types
-binomlogpdf(n::Real, p::Real, k::Real) = -log1p(n) - logbeta(n - k + 1, k + 1) + k * log(p) + (n - k) * log1p(-p)
+binomlogpdf(n::Real, p::Real, k::Real) = betalogpdf(k + 1, n - k + 1, p) - log(n + 1)
+
+binomcdf(n::Real, p::Real, k::Real) = betaccdf(k + 1, n - k, p)
+
+binomccdf(n::Real, p::Real, k::Real) = betacdf(k + 1, n - k, p)
+
+binomlogcdf(n::Real, p::Real, k::Real) = betalogccdf(k + 1, n - k, p)
+
+binomlogccdf(n::Real, p::Real, k::Real) = betalogcdf(k + 1, n - k, p)
```

Comment:
We'd then have to write the methods for the log versions again here and since the pattern is the same for the log and non-log versions, I think current version is preferable. If needed, I can add a comment.